examples/pipeline.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       platform: linux
       image_resource:
         type: docker-image
-        source: {repository: machinerytool/osc_opensuse42.1}
+        source: {repository: machinerytool/osc_opensuse_leap}
```

Comment:
I wonder why we didn't follow the common docker image versioning for this. E.g. machinerytool/osc_opensuse:42.1 etc.
This is not relevant to this PR though.